rules/windows/process_creation/win_anydesk_silent_install.yml

```diff
@@ -0,0 +1,31 @@
+title: AnyDesk Silent Installation
+id: 114e7f1c-f137-48c8-8f54-3088c24ce4b9
+status: experimental
+author: Ján Trenčanský
+date: 2021/08/06
+description: AnyDesk Remote Desktop silent installation can be used by attacker to gain remote access.
+references:
+    - https://twitter.com/TheDFIRReport/status/1423361119926816776?s=20
+    - https://support.anydesk.com/Automatic_Deployment
+tags:
+    - attack.t1219
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_anydesk:
+        CommandLine|contains: 'anydesk.exe'
+    selection_install:
+        CommandLine|contains: '--install'
+    selection_with_win:
+        CommandLine|contains: '--start-with-win'
+    selection_silent:
+        CommandLine|contains: '--silent'
+    condition: all of them
```

Comment:
Why not
```yaml
selection:
    Product: AnyDesk
    CommandLine|contains|all:
        '--install '
        '--start-with-win '
        '--silent '
condition: selection
```

Reply:
I'd create two sections since not all log sources contain the OriginalFileName

Reply:
--start-with-win is pretty specific - a quick Google search didn't return many FPs. Combined with the other two it should be good enough.

Reply:
I had the OriginalFilename field empty for AnyDesk executable.  I did want to use the Product field, but noticed it's not mapped in some outputs. Two sections are probably the best way to go.